CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## 0.0.38 - 2020-02-27
 - Doc updates (migrated from legacy API)
 
+## 0.0.38-fb-queryMetadataEditor-0.0 - 2020-02-27
+- Item 6083: QueryMetadata Editor migration
```

Comment:
nit: Consider rewording release comment to be more contextually relevant to labkey-api-js:

```
- Item 6083: Add Query.getQueryMetadata endpoint
```